### PR TITLE
Highlight supported Prometheus integration on top

### PR DESCRIPTION
**Pull request by amujumdar**
Google Cloud Monitoring develops and maintains stackdriver-prometheus to for GKE users. 
It is well supported and designed to meet expectations of Prometheus users. Many GKE
users are confused by the current README, use Prometheus integration from this repository
and run into issues. Highlighting this on top clears the confusion.

### CODE REVIEW

File: prometheus-to-sd/README.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+# :exclamation: Note
+
+**For collecting Prometheus metrics on Google Kubernetes Engine (GKE) clusers, use Google Cloud Monitoring
```
Comment (xichen2020) [inline prometheus-to-sd/README.md:3]: nit: Link to Google cloud monitoring?

Comment (xichen2020) [reply prometheus-to-sd/README.md:3]: Also typo: `clusters`

**Comment by josephflynch** [comment]
LGTM

File: prometheus-to-sd/README.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
+# :exclamation: Note 
+
+**For collecting Prometheus metrics on Google Kubernetes Engine (GKE) clusters, use
+[Google Cloud Monitoring][cloudMonitoringSite] (fka Stackdriver) supported integration.**
```
Comment (mtwo) [inline prometheus-to-sd/README.md:4]:
```suggestion
[Google Cloud Monitoring's][cloudMonitoringSite] (fka Stackdriver) supported integration.**
```

File: prometheus-to-sd/README.md [hunk 3]
```diff
@@ -1,3 +1,15 @@
...
+- [Documentation][cloudMonitoringPromSite]
+- [Source Code][cloudMonitoringPromCode]
+
+[Google Cloud Monitoring][cloudMonitoringSite] supported [integration][cloudMonitoringPromSite] is designed to meet the expectations of Prometheus users and to make it easy to run with Prometheus server. It is intended to monitor all your applications, Kubernetes and beyond.
```
Comment (mtwo) [inline prometheus-to-sd/README.md:8]:
```suggestion
[Google Cloud Monitoring's][cloudMonitoringSite] supported [integration][cloudMonitoringPromSite] is designed to meet the expectations of Prometheus users and to make it easy to run with Prometheus server. It is intended to monitor all your applications, Kubernetes and beyond.
```

**Comment by mtwo** [comment]
Made two grammatical suggestions, approved otherwise

**Comment by 44past4** [comment]
LGTM

**Pull request merged**